frame/contracts/Cargo.toml

```diff
@@ -54,6 +54,9 @@ env_logger = "0.9"
 pretty_assertions = "1"
 wat = "1"
 
+# Include this package, so that `unsafe-debug` feature is included for testing.
+pallet-contracts = { path = ".", default-features = false, features = ["unsafe-debug"] }
```

Comment:
No. Just conditionally compile the tests.

Reply:
so this will never run in CI? which makes the test less useful..

Reply:
Doesn't the CI run `cargo test --all-features`?

Reply:
🤔  I thought not.
Some other folks are doing similar things 
https://github.com/paritytech/polkadot/pull/7538/files/b72c655f9323004d8da523f6f4690204172e9b83#r1275964591

Reply:
The proper solution is to change the CI script to enable all features. This just doesn't seem like intended behaviour.

Reply:
I'm not sure where how I should change it then :|

Reply:
added; the corresponing CI job succeeded, but I have no proof in logs for running this particular testcase - the log is truncated

but to be 100% I will corrupt an assertion in the testcase and see whether it is reported

Reply:
confirmed: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/3324722

reverting change then; @athei @pgherveou so I guess we are done here